pandas/tests/categorical/test_missing.py

```diff
@@ -53,3 +53,23 @@ def test_set_item_nan(self):
 
         exp = Categorical([1, np.nan, 3], categories=[1, 2, 3])
         tm.assert_categorical_equal(cat, exp)
+
+    def test_fillna_raises(self):
+        # https://github.com/pandas-dev/pandas/issues/19682
+
+        cat = Categorical([1, 2, 3])
+
+        xpr = "Cannot specify both 'value' and 'method'."
+
```

Comment:
can you remove the line between the message and the condition, personal preference but its slightly distracting